src/families/cosmos/libcore-signOperation.js

```diff
@@ -61,6 +61,14 @@ async function signTransaction({
       ? "REWARD"
       : "OUT";
 
+  const extra = {};
+  if (transaction.mode === "REDELETE") {
```

Comment:
s/redelete/redelegate ?

Reply:
good catch! i'll try to add test in bot too.